src/ducks/myApps/index.js

```diff
@@ -39,60 +47,118 @@ const isFetching = (state = false, action) => {
 
 export const error = (state = null, action) => {
   switch (action.type) {
-    case FETCH_MY_APPS_FAILURE:
+    case FETCH_APPS_FAILURE:
     case UNINSTALL_APP_FAILURE:
       return action.error
     default:
       return state
   }
 }
 
-export const myAppsReducers = combineReducers({
+export const appsReducers = combineReducers({
   list,
   error,
   isFetching
 })
 
-async function getIcon (url) {
+export function getInstalledApps (state) {
+  return state.apps.list.filter(app => app.installed)
+}
+
+async function _getIcon (url) {
+  if (!url) return ''
   const icon = await cozy.client.fetchJSON('GET', url)
 
   try {
     return 'data:image/svg+xml;base64,' + btoa(icon)
   } catch (e) { // eslint-disable-line
-    return URL.createObjectURL(icon)
+    try {
+      return URL.createObjectURL(icon)
+    } catch (e) {
+      return ''
+    }
   }
 }
 
-const NOT_REMOVABLE_APPS = ['drive', 'collect']
-const NOT_DISPLAYED_APPS = ['settings', 'store', 'onboarding']
+function _consolidateApps (stateApps, newAppsInfos) {
```

Comment:
I see your point of using an `_` prefix here, for indicating that this method is private, bet if I am right it is not a convention we are using in our other apps. So to be coherent, I suggest to remove the `_` what do you think ?

Reply:
It's not only because it's private, I use it also to say explicitly that these methods are not supposed to be exported and used externally like that. If this case happens, that parts should be moved in a external library/file.